sdk/python/foundation-models/llama2/promptFlow/standard-flow-sample/Fligh_fare/flow.meta.yaml

```diff
@@ -0,0 +1,10 @@
+$schema: https://azuremlschemas.azureedge.net/latest/flow.schema.json
+name: template_standard_flow
+display_name: Template Standard Flow
```

Comment:
Change the display name for a more accurate name

Reply:
Done